.github/contributing.md

```diff
@@ -2,6 +2,53 @@
 
 First of all, we are so glad that you are reading this right now, we need YOU to make [Amadeus for Developers documentation](https://amadeus4dev.github.io/developer-guides/) more consistent, and readable, adding missing information, correcting factual errors, or/and fixing typos. :heart_eyes::tada: 
 
+### Style guide
+
+Our guides are written in the second-person point of view, which is more engaging for the reader.
+
+#### Menu item names and headers
+
+Keep the menu item names as short as possible but describe them in the page headers. For example, **Pricing > Pricing options for Amadeus Travel APIs**.
+
+#### Start each step with a verb
+
+When writing step-by-step instructions, start each step with a verb. For example:
+
+1. Create an account.
+2. Register your app.
+3. Make an API call.
+
+#### Information and warning boxes
+
+!!! warning
+    This text box is used to describe an action that can potentially inflict damage to your system or business.
+
+!!!information
```

Comment:
For the contributor's guide can we choose between: `!!!warning !!!information`  and `!!! warning !!! information` for consistency? Even though both ways work fine. Personally I prefer with the space after the !!!, it looks prettier :P

Reply:
Well spotted, Anna, thanks!